hadoop-tools/hadoop-distcp/src/site/markdown/DistCp.md.vm

```diff
@@ -349,7 +349,7 @@ Command Line Options
 | `-filters` | The path to a file containing a list of pattern strings, one string per line, such that paths matching the pattern will be excluded from the copy. | Support regular expressions specified by java.util.regex.Pattern. |
 | `-filelimit <n>` | Limit the total number of files to be <= n | **Deprecated!** Ignored in the new DistCp. |
 | `-sizelimit <n>` | Limit the total size to be <= n bytes | **Deprecated!** Ignored in the new DistCp. |
-| `-delete` | Delete the files existing in the dst but not in src | The deletion is done by FS Shell. So the trash will be used, if it is enable. Delete is applicable only with update or overwrite options. |
+| `-delete [-useTrash]` | Delete the file`[-useTrash]` is enabled,the files will be moved to trash. Notice that `[-useTrash]` option on some object store does a copy and delete ops and costs O(Num). Delete is applicable only with update or overwrite options. |
```

Comment:
1.  add space between , and "the"
2. for cost of object stores, just say "can be slow"